crates/jstzd/src/task/octez_baker.rs

```diff
@@ -0,0 +1,215 @@
+use super::{
+    child_wrapper::{ChildWrapper, SharedChildWrapper},
+    endpoint::Endpoint,
+    octez_client::OctezClient,
+    Task,
+};
+use anyhow::{anyhow, Result};
+use async_trait::async_trait;
+use http::Uri;
+use octez::OctezNodeConfig;
+
+use std::{fmt::Display, path::PathBuf, str::FromStr};
+use tokio::process::Command;
+
+#[derive(PartialEq, Debug, Clone)]
+pub enum Protocol {
```

Comment:
You can add the protocol versions to enum `Protocol` in `crates/jstzd/src/protocol/mod.rs` and use that enum, though the protocol version-baker executable name mapping needs to be revised.